packages/core-mobile/app/new/features/tokenManagement/store.ts

```diff
@@ -0,0 +1,6 @@
+import { Network } from '@avalabs/core-chains-sdk'
+import { createZustandStore } from 'common/utils/createZustandStore'
+
+export const useTokenAddress = createZustandStore<string>('')
+
+export const useNetwork = createZustandStore<Network | undefined>(undefined)
```

Comment:
could we rename this to useSelectedNetwork? I thought this was part of our useNetworks hook at first.